src/apis/issue.ts

```diff
@@ -114,6 +114,15 @@ export class DefaultIssueAPI implements IssueAPI {
         return issueRequests.map((v) => v[1]);
     }
 
+    async mapForUser(account: AccountId): Promise<Map<H256, IssueRequest>> {
```

Comment:
I would prefer to explicitly state what the map key is in the function name. `indexedByIdForUser` or something like that?